src/codeu/chat/client/android/app/src/main/java/com/google/codeu/chatme/model/Conversation.java

```diff
@@ -0,0 +1,53 @@
+package com.google.codeu.chatme.model;
+
+import com.google.firebase.database.IgnoreExtraProperties;
+
+import java.util.ArrayList;
+import java.util.List;
+
+@IgnoreExtraProperties
+public final class Conversation {
+
+    public String id;
+    public String owner;
+    public long timeCreated;
+
+    private final List<String> participants = new ArrayList<>();
+
+    public Conversation() {
+    }
+
+    public Conversation(String owner) {
+        this.owner = owner;
+        this.participants.add(owner);
+        this.timeCreated = System.currentTimeMillis();
+    }
+
+    public String getId() {
+        return id;
+    }
+
+    public void setId(String id) {
+        this.id = id;
+    }
+
+    public String getOwner() {
+        return owner;
+    }
+
+    public void setOwner(String owner) {
+        this.owner = owner;
+    }
+
+    public long getTimeCreated() {
+        return timeCreated;
+    }
+
+    public void setTimeCreated(long timeCreated) {
+        this.timeCreated = timeCreated;
+    }
+
+    public List<String> getParticipants() {
```

Comment:
Should this be an immutable list with a getter and setter like the other fields?  Using a "live" list here may lead to concurrency bugs.  If you choose to keep this API, please add a Javadoc comment explaining that the list returned is a reference to mutable internal state.